MotorDriverInfo[] motor_driver_info